src/pylada/vasp/keywords.py

```diff
@@ -874,7 +874,11 @@ def output_map(self, **kwargs):
         from ..error import ValueError
         from ..crystal import write, read, specieset
         from . import files
-
+        try:
+            from pylada import is_vasp_4
+        except:
+            is_vasp_4 = False
+        
```

Comment:
I think it would be better to simply add `is_vasp_4 = False` in config/vasp.py and then simply do

```suggestion
        from pylada import is_vasp_4
```

Looking back, that's what we should have done in the first place!